net-square-designs/Errnd
# Create users profile

#### What does this PR do?
It creates and views a user profile
#### Description of Task to be completed
- create a users profile
- view a users profile
#### How should this be manually tested?
* Clone the repo
* Execute the following commands on your terminal

> * git checkout ` ft-sign-up-a-user`.
> * npm install
> * npm run start:dev
> * using postman navigate to **POST** `http://localhost:3005/api/v1/profile/:username`.
> * add this json in the raw filed `{
"firstName": "John",
"lastName": "Doe",
"image": "",
"location": "Lagos",
"bio": "I am a sofware engineer",
"phone": "`
> * click send
> * You should see a response body containing the message `Users profile created successfullyy`
> * using postman navigate to **GET** `http://localhost:3005/api/v1/profile/:username`.§
> * click send
> * You should see a response body containing the message `Users profile returned succesfully`
#### Any background context you want to provide?
Previously, there was no way to create or view a users profile

#### Screenshots (if appropriate)
N/A
